src/plugin-host/plugin-events/event-arguments/no-arguments-plugin-event-arguments.ts

```diff
@@ -0,0 +1,17 @@
+import { PluginEventArguments } from "../plugin-event-arguments";
+import { Chat } from "../../../chat/chat";
+
+/**
+ * Leaderboard Reset Plugin Event Arguments.
```

Comment:
These are copy-pasted comments and do not belong in this file.

Reply:
Fixed